package.json

```diff
@@ -19,19 +19,32 @@
     "test:smoke:test": "mkdir -p build/reports && cross-env TESTS=smoke cucumber-js src/test/smoke/features/ --require 'src/test/common/steps/*.js' --require 'src/test/smoke/config' -f json:build/reports/smoke-report.json",
     "test:smoke:report": "node src/test/smoke/smoke-report.js",
-    "test:compatibility": "mkdir -p build/reports && cross-env TESTS=compatibility grunt compatibility",
-    "test:compatibility:debug": "cross-env TESTS=compatibility node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
-    "test:compatibility:report": "node src/test/compatibility/compatibility-report.js",
     "test:compatibility:local": "npm-run-all -s test:compatibility:local:test test:compatibility:local:report",
-    "test:compatibility:local:test": "mkdir -p build/reports && cross-env TESTS=localCompatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
-    "test:compatibility:local:report": "node src/test/compatibility/compatibility-report.js",
+    "test:compatibility:local:test": "mkdir -p build/reports/compatibility && cross-env TESTS=localCompatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
+    "test:compatibility:local:report": "node src/test/compatibility/local-compatibility-report.js",
     "test:accessibility": "node 'src/test/a11y/test-suite.js'",
     "test:browser": "npm run test:wait && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report",
     "test:wait": "./npm_scripts/wait_for_app_to_start.sh",
     "coverage": "nyc report --reporter=lcov --reporter=text-lcov > coverage.lcov && codecov",
     "build": "npm run build:sass",
     "build:sass": "node-sass node_modules/govuk-frontend/all.scss src/web/assets/styles/all.css",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "test:compatibility": "mkdir -p build/reports/compatibility && cross-env TESTS=compatibility npm-run-all -c -p --max-parallel 1 compat:win compat:osx compat:mobile",
+    "test:compatibility:debug": "node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' --format ./src/test/compatibility/custom-json-formatter.js:build/reports/compatibility-report.json",
+    "test:compatibility:report": "node src/test/compatibility/compatibility-report.js",
+    "compat:win": "cross-env BROWSER_STACK_OS=Windows BROWSER_STACK_OS_VERSION=10 npm-run-all -c -p --max-parallel 5 compat:win:ie compat:win:edge compat:win:chrome compat:win:firefox",
```

Comment:
within each group of compatibility tests, tests run in parallel (up to 5 at a time, as this is the most that can run in parallel on BrowserStack)